python/golem_task_api/client.py

```diff
@@ -43,7 +44,7 @@ def spawn_server(self, command: str, port: int) -> Tuple[str, int]:
         pass
 
     @abc.abstractmethod
-    async def wait_after_shutdown(self) -> None:
+    async def wait_for_shutdown(self) -> None:
```

Comment:
I'd interpret `wait_for_shutdown` as waiting for the `Shutdown` request which isn't the point of this method.
I understand that maybe a better name is needed here but `wait_for_shutdown` is not it imo.

Reply:
maybe `wait_until_exit` or smiliar

Reply:
```suggestion
    async def wait_for_shutdown_completed(self) -> None:
```

Reply:
I thought the default reading is `wait_for_SERVICE_shutdown` since this a method of the service class but since we have a `Shutdown` request this indeed can be confusing. Maybe `wait_until_shutdown_complete` then?

Reply:
haha, was thinking that as well, left `_for` since i could not decide :P

Reply:
Renamed.